sdk/batch/azure-batch/azure/batch/custom/patch.py

```diff
@@ -1,311 +0,0 @@
-                self.tasks_to_add.extendleft(chunk_tasks_to_add)
-                # Unknown State - don't know if tasks failed to add or were successful
-                self.errors.appendleft(e)
-        except Exception as e:  # pylint: disable=broad-except
-            # Re-add to pending queue as unknown status / don't have result
-            self.tasks_to_add.extendleft(chunk_tasks_to_add)
-            # Unknown State - don't know if tasks failed to add or were successful
-            self.errors.appendleft(e)
-        else:
-            try:
-                add_collection_response = add_collection_response.output
-            except AttributeError:
-                pass
-
-            for task_result in add_collection_response.value:  # pylint: disable=no-member
-                if task_result.status == TaskAddStatus.server_error:
-                    # Server error will be retried
-                    with self._pending_queue_lock:
-                        for task in chunk_tasks_to_add:
-                            if task.id == task_result.task_id:
-                                self.tasks_to_add.appendleft(task)
-                elif (task_result.status == TaskAddStatus.client_error
-                        and not task_result.error.code == "TaskExists"):
-                    # Client error will be recorded unless Task already exists
-                    self.failure_tasks.appendleft(task_result)
-                else:
-                    results_queue.appendleft(task_result)
-
-    def task_collection_thread_handler(self, results_queue):
-        """Main method for worker to run
-
-        Pops a chunk of tasks off the collection of pending tasks to be added and submits them to be added.
-
-        :param collections.deque results_queue: Queue for worker to output results to
-        """
-        # Add tasks until either we run out or we run into an unexpected error
-        while self.tasks_to_add and not self.errors:
-            max_tasks = self._max_tasks_per_request  # local copy
-            chunk_tasks_to_add = []
-            with self._pending_queue_lock:
-                while len(chunk_tasks_to_add) < max_tasks and self.tasks_to_add:
-                    chunk_tasks_to_add.append(self.tasks_to_add.pop())
-
-            if chunk_tasks_to_add:
-                self._bulk_add_tasks(results_queue, chunk_tasks_to_add)
-
-
-def _handle_output(results_queue):
-    """Scan output for exceptions
-
-    If there is an output from an add task collection call add it to the results.
-
-    :param results_queue: Queue containing results of attempted add_collection's
-    :type results_queue: collections.deque
-    :return: list of TaskAddResults
-    :rtype: list[~TaskAddResult]
-    """
-    results = []
-    while results_queue:
-        queue_item = results_queue.pop()
-        results.append(queue_item)
-    return results
-
-
-def build_new_add_collection(original_add_collection):
-    def bulk_add_collection(
```

Comment:
has this been moved or are we not implementing it in track2

Reply:
It's renamed as `create_task_collections` https://github.com/Azure/azure-sdk-for-python/blob/85cabcff4e3dcac7b21e4d113cb5fcc719f8d24f/sdk/batch/azure-batch/azure/batch/_operations/_patch.py#L33-L43